priv/chttpd_handler.cfg

```diff
@@ -0,0 +1,47 @@
+%% Licensed under the Apache License,  Version 2.0 (the "License");  you may not
+%% use this file except in compliance with the License. You may obtain a copy of
+%% the License at
+%%
+%%                 http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless  required  by  applicable  law  or  agreed  to  in  writing,  software
+%% distributed  under the  License  is distributed on an "AS IS" BASIS,  WITHOUT
+%% WARRANTIES  OR  CONDITIONS  OF  ANY  KIND, either express or implied. See the
+%% License for the specific language governing permissions and limitations under
+%% the License.
+
+%% @doc link endpoints to callbacks - configs of multiple deps are combined.
+
+{docs, clauses, [
```

Comment:
As far as I understood, first element of this config structure defined to what API level following resources are belong: db for /db/:resource, design for /db/_design/ddoc/:resource etc. But why server root level is named as docs then? This causes false assumption that these resources will be available at /db/docid/ level.